ts/editor/ClozeButton.svelte

```diff
@@ -5,20 +5,29 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
 <script lang="typescript">
     import IconButton from "../components/IconButton.svelte";
     import WithShortcut from "../components/WithShortcut.svelte";
-    import OnlyEditable from "./OnlyEditable.svelte";
 
     import * as tr from "../lib/ftl";
     import { withButton } from "../components/helpers";
     import { ellipseIcon } from "./icons";
-    import { forEditorField } from ".";
-    import { wrapCurrent } from "./wrap";
+    import { get } from "svelte/store";
+    import {
+        getContext,
+        noteEditorKey,
+        activeInputKey,
+        focusInEditableKey,
+    } from "./context";
+
+    const noteEditor = getContext(noteEditorKey);
+    const activeInput = getContext(activeInputKey);
```

Comment:
"scopedProperty" could work too

Reply:
It sounds good. What is nice about `context.ts`, that it would have been very straightforward to export for add-ons using `registerPackage`. But I guess also register those individual "context getters"

Reply:
Ok, #1434 should fix these. After rebasing over main, you'll need to add the deps to editable/editor (https://github.com/ankitects/anki/pull/1434/commits/bb5053990b86e0499afe7eebf358de9ce0dabb4d), and then build ts/...

You may also need to cmd+shift+p, Svelte: Restart Language Server after doing a build

If you spot further problems, please let me know.